replica/tablets.cc

```diff
@@ -556,11 +568,11 @@ class tablet_incremental_selector : public sstables::incremental_selector_impl {
         if (!_cur_set || pos.token() >= _lowest_next_token) {
             auto idx = _tset.group_of(token);
             if (!token.is_maximum()) {
-                _cur_set = _tset._sstable_sets[idx];
+                _cur_set = _tset.find_sstable_set(idx);
```

Comment:
I hope that the assumption that the search token must be covered by an allocated tablet holds water.